Analytics/resources/test_request_for_data.py

```diff
@@ -0,0 +1,99 @@
+# import pytest
+import unittest
```

Comment:
Are all of these import either python imports, 3rd party, or local? If not please separate them